### ISSUE TITLE
Collected accessibility data should be uploaded more frequently

### ISSUE THREAD

**Issue by kotarohara** [ProjectSidewalk/SidewalkWebpage#44]
We should upload the collected accessibility data more frequently. Currently, we send the data when (i) a user reaches the end of the current street segment s/he is auditing or (ii) after after walking long distance. This prevents us from collecting data from users who uses the system for a very short amount of time.

**Comment by kotarohara** [comment]
I actually do submit data quite frequently; after every 30 interactions (e.g., walking, panning, labeling), data is submitted to the server.

**Comment by jonfroehlich** [comment]
Is this enough? How much data might we lose here? We absolutely want to minimize losing data... perhaps we can, at the very least, upload after every label (plus have a frequency check if the user has done 15-30 interactions with no upload, then this triggers an upload).

**Comment by kotarohara** [comment]
I forgot to close this issue when I closed #71 . I'm closing it. I also made change so closing browser window and moving to different page from the auditing page triggers a data submission.

**Comment by jonfroehlich** [comment]
This is great. Thanks!
